src/main/java/com/prototype/booking/api/referencedata/RoomsInfo.java

```diff
@@ -0,0 +1,31 @@
+package com.prototype.booking.api.referencedata;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor @AllArgsConstructor
+@Entity
+@Table(name ="MEETING_ROOM")
+public class RoomsInfo {
```

Comment:
rename the entity class to **Room** or **MeetingRoom **?

Reply:
MeetingRoom should be ok